src/renderer/rendererutils.ts

```diff
@@ -26,24 +7,16 @@
 
     // Not sure how we can decide who won or lost yet. 
     // Combat log doesn't make it obvious.
-    if ((category == "Battlegrounds") || (category == "Solo Shuffle")) {
```

Comment:
We've lost this logic as far as I can tell?

Reply:
It's no longer neccessary. See `getResultText()`, default switch case takes care of that.